app/controllers/admin/orders_controller.rb

```diff
@@ -0,0 +1,38 @@
+class Admin::OrdersController < ApplicationController
+  # before_action :require_admin
+  before_action :find_order, except: [:index]
+
+  def index
+    @orders = Order.all
+  end
+
+  def edit
+  end
+
+  def update
+    if @order.update(order_params)
+      redirect_to
+    else
+      render 'Edit'
+    end 
```

Comment:
Great job on Admin::OrdersController, Hadi!! Think you may have missed the redirection here haha. Also, I didn't know rendering the view could work as a string.